cloud-account/join-or-leave-an-organization.md

```diff
@@ -6,11 +6,30 @@ mapped_urls:
 
 # Join or leave an organization
 
-% What needs to be done: Refine
+## Accept an invitation [ec-accept-invitation]
 
-% Scope notes: https://github.com/elastic/docs-projects/issues/304
+Invitations to join an organization are sent by email. Invited users have 72 hours to accept the invitation. If they do not join within that period, an administrator of the organization will have to send a new invitation. Refer to [manage users](/deploy-manage/users-roles/cloud-organization/manage-users.md) for more information.
 
-% Use migrated content from existing pages that map to this page:
+## Leave an organization [ec-leave-organization]
 
-% - [ ] ./raw-migrated-files/cloud/cloud/ec-invite-users.md
-% - [ ] ./raw-migrated-files/docs-content/serverless/general-manage-organization.md
\ No newline at end of file
+On the **Members** tab of the **Organization** page, click the three dots corresponding to your email address and select **Leave organization**.
+
+If you’re the only user in the organization, you can only leave if you deleted all your deployments and projects, and you don’t have pending bills.
+
+## Join an organization from an existing {{ecloud}} account [ec-join-invitation]
+
+You already belong to an organization. If you want to join a new one and bring your deployments over, follow these steps:
```

Comment:
I know this copy was from the original, but it's confusing. why does the person already belong to an org?

Reply:
It's a use case where a user might want to change its own organization, so the user already exists and belongs to an organization. I'll rephrase a bit the content (it was a copy from the original, yes), let's see how it works...

Reply:
I think it's now clear and covers deployments and projects.